### ISSUE TITLE
Unicode in HoloMap labels

### ISSUE THREAD

**Issue by vascotenner** [ioam/holoviews#350]
HoloMaps do not render properly when the labels of the items are unicode.

Works:
```python
dim_frequency = hv.Dimension('Frequency', unit='Hz')
dim_powerspectrum = hv.Dimension('Power density', unit='dB/Hz', range=(-100,0))
hv.HoloMap({str(chr(i+65)+'a'): hv.Curve(([1,2,3],np.array([1,2,3])**i), kdims=[dim_frequency], 
                                vdims=[dim_powerspectrum]) for i in range(10)}, kdims=['Direction'])
```

Does not work:
```python
dim_frequency = hv.Dimension('Frequency', unit='Hz')
dim_powerspectrum = hv.Dimension('Power density', unit='dB/Hz', range=(-100,0))
hv.HoloMap({unicode(chr(i+65)+'a'): hv.Curve(([1,2,3],np.array([1,2,3])**i), kdims=[dim_frequency], 
                                vdims=[dim_powerspectrum]) for i in range(10)}, kdims=['Direction'])
```

```
Javascript error adding output!
SyntaxError: Unexpected string
See your browser Javascript console for more details.
```

Javascript console:
```
SyntaxError: Unexpected string(…)   outputarea.js:415
````

**Comment by jlstevens** [comment]
Thanks for reporting this!

I see no reason why our javascript code should choke on unicode strings. Hopefully the fix will be relatively straight forward...

**Comment by philippjfr** [comment]
This has been fixed in #376. Closing.
